### ISSUE TITLE
Install Script failed

### ISSUE THREAD

**Issue by CombinedNetworks** [nextcloud/vm#2542]
### Steps To Reproduce

1) Fresh install of Ubuntu 22.04 LTS including all Updates
2) wget https://raw.githubusercontent.com/nextcloud/vm/master/nextcloud_install_production.sh
3) bash nextcloud_install_production.sh

### Expected Result

Install script work just to this point of installing nextcloud
...
...
...
...
Fetching the latest Nextcloud version...
27.0.2
Downloading nextcloud-27.0.2...
Testing if network is OK...
Checking connection...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  161M  100  161M    0     0  22.0M      0  0:00:07  0:00:07 --:--:-- 21.8M
Checking SHA256 checksum...
nextcloud-27.0.2.tar.bz2: OK
Checking GPG integrity...
gpg: directory '/root/.gnupg' created
gpg: keybox '/root/.gnupg/pubring.kbx' created
gpg: keyserver receive failed: Connection timed out
gpg: Signature made Di 08 Aug 2023 08:26:21 UTC
gpg:                using RSA key 28806A878AE423A28372792ED75899B9A724937A
gpg: Can't check signature: No public key
[⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻⣾⣽⣻]
[⣾⣽⣻⣾⣽⣻Setting secure permissions...
Creating possible missing Directories
chmod Files and Directories
chown Directories
chmod/chown .htaccess
]
Installing Nextcloud...
Nextcloud is not installed - only a limited number of commands are available

We stopp it with CTRL-C

sudo -u www-data php /var/www/nextcloud/occ maintenance:install --data-dir=/mnt/                                                                     ncdata --database=pgsql --database-name=nextcloud_db --database-user=nextcloud_d                                                                     b_user --database-pass=3JE6C7wesN9yZz81KPb6CdNyi6f2 --admin-user=ncadmin --admin                                                                     -pass=nextcloud failed

Nextcloud is not installed - only a limited number of commands are available
^CNextcloud is not installed - only a limited number of commands are available
Nextcloud is not installed - only a limited number of commands are available
^CNextcloud is not installed - only a limited number of commands are available

  There are no commands defined in the "app" namespace.

### Actual Result

No nextcloud is installed!

### Build Version

27.0.2

### Environment

By using the scripts

### Environment Details

Ubuntu Server 22.04.1 LTS

**Comment by enoch85** [comment]
> We stopp it with CTRL-C

Please be patient. It takes a while.

**Comment by lenonmoreschi** [comment]
Yes, the same is happening to me. I waited for more than 2h30min for this step of the process before pressing Ctrl+C. The error looks exactly like yours.

**Comment by enoch85** [comment]
I've done 5 installations today, they took longer than expected, but they worked. Just leave it over night and see what happens 

I think it's a new bug in NC 27, have investigated though.

**Comment by lenonmoreschi** [comment]
ok, thanks for the clarification! I will continue testing as well.

**Comment by shylaev** [comment]
> I've done 5 installations today, they took longer than expected, but they worked. Just leave it over night and see what happens

I tried several times, once even waited over six hours, but to no avail.
> 
> I think it's a new bug in NC 27, have investigated though.

![image](https://github.com/nextcloud/vm/assets/93459981/8a42a387-bb02-4337-be37-25afa5f717a5)

version 26.0.5 has the same issue. don't mention GPG's check because that instance is behind a firewall

**Comment by shylaev** [comment]
well. i think i found the solution:
in line 547 of nextcloud_install_production.sh script uses function "nextcloud_occ()" from lib.sh.
change it to "sudo -u www-data php "$NCPATH"/occ" instead nextcloud_occ.
it work for me

**Comment by CombinedNetworks** [comment]
Thanks to shylaevI
This works now for me!

**Comment by enoch85** [comment]
This is now in master. Thanks for the feedback!

**Comment by lenonmoreschi** [comment]
This is perfect! Thanks for the feedback, it worked for me too. Thank you all.
